zdm/scripts/plot_pzdm_grid.py

```diff
@@ -23,7 +23,7 @@
 def main():
 
```

Comment:
general comment: this script seems to have been altered for just specific purposes. In general, don't commit changes to such files which are simply minor alterations for testing purposes. In any case, what's here is fine, since I think I need to make a "simple" and "complex" version of this, and I'd left a bunch of annoying functionality in this to begin with, i.e. I hadn't followed my own advice!